packages/modules/create-imodel/src/components/create-imodel/CreateIModel.tsx

```diff
@@ -52,8 +52,8 @@ export type CreateIModelProps = {
     /** Longitude label. */
     longitude?: string;
   };
-  /** ProjectId where to create an iModel. */
-  projectId: string;
+  /** iTwinId where to create an iModel. */
+  iTwinId: string;
```

Comment:
Just to be sure, this will come with the new major release as its breaking change, right?
If so, you need to update the changelog with the exact changes you have done as this is more important to the user than what api is being used

Reply:
Yes these changes along with other changes which are present in next/browser branch will be a major release. Updated changelogs